### ISSUE TITLE
Fix tests

### ISSUE THREAD

**Issue by ahmetalpbalkan** [aspnet/aspnet-docker#49]
Apparently the directory structure at aspnet/Home/Samples has changed. Therefore tests are expected to fail until we adapt.

**Comment by Eilon** [comment]
@glennc can perhaps offer advice on this if needed. Though for the most part the new structure should be fairly obvious. Sorry for breaking everything!

**Comment by vlesierse** [comment]
I was thinking about an approach to try `samples\$VERSION` first and fall back to `git chechout v$VERSION` when the folder doesn't exist.

**Comment by ahmetalpbalkan** [comment]
@vlesierse we can do something like:

    ... | grep -v 'beta[1-3]'

**Comment by vlesierse** [comment]
Excluding beta 1 to 3 from the CI? Makes sense... Those images aren't very likely to change.

**Comment by vlesierse** [comment]
Perhaps this would be more future proof and excludes the `vs-1.0.0-beta4` and the `coreclr-1.0.0-beta5-11624` as well for the time being.
```
find . -path ./.git -prune -o -name Dockerfile -print0 | xargs -0 -n1 dirname | sed -e "s/\.\///" | grep -E '^([0-9])(\.[0-9]){1,2}(-.+)?$' | grep -v '1.0.0-beta[1-3]'
```

**Comment by ahmetalpbalkan** [comment]
@vlesierse not sure what the command above does. :smile: we don't have to be super reliable as the new versions come b/c apparently aspnet/Home could break, too. :) Let's just make sure we just grab all dockerfiles except earlier ones and avoid false positives in the CI.

**Comment by ahmetalpbalkan** [comment]
closed by #57
